app.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs')
+const fetch = require('node-fetch')
+const colors = require('colors')
+
+console.log("Natalia Olmos / Proyecto MDLINKS /Laboratoria CDMX009")
+
+function getContentString() {
+
+    let index = process.argv.indexOf("--file")
+    if (index < 0) return console.log("You need to use a valid uri flag --file")
+    let uri = process.argv[index + 1]
+    let string = fs.readFileSync(uri, 'utf8')
+    return string
+}
```

Comment:
mejorar la definición
```suggestion
function getContentPath() {

    let index = process.argv.indexOf("--file")
    if (index < 0) return console.log("You need to use a valid uri flag --file")
    let path = process.argv[index + 1]
    let file = fs.readFileSync(uri, 'utf8')
    return file
}
```